doc/application/application.rst

```diff
@@ -98,8 +109,26 @@ Follow these steps to create a new application directory. (Refer to
       cd app
       mkdir src
 
-#. Create a :file:`CMakeLists.txt` file in your application directory with the
-   following contents:
+#. Create an empty :file:`CMakeLists.txt` file in your application directory.
+
+#. Optionally include additional :file:`.conf` files in your
+   :file:`CMakeLists.txt`. By default the build system will pick up
+   :file:`prj.conf` automatically:
+
+   .. code-block:: cmake
+
+   set(CONF_FILE "prj.conf ${APPLICATION_SOURCE_DIR}/boards/${BOARD}.conf")
+
+#. Optionally include a reference to your :file:`dts.overlay` in your
+   :file:`CMakeLists.txt`. :file:`.overlay` files named after the board
+   (:file:`<board>.overlay`) will be picked up automatically by the build
+   system:
+
+   .. code-block:: cmake
+
+   set(DTC_OVERLAY_FILE "${APPLICATION_SOURCE_DIR}/dts.overlay")
+
+#. Include the :file:`boilerplate.cmake` to pull in the Zephyr build system:
```

Comment:
This will read strangely in the rendered HTML. How about:

```
#. Include the :file:`boilerplate.cmake` file provided by Zephyr to integrate with its build system:
```

(Notice the addition of the extra word "file" in what gets rendered as HTML)